jahia/jahia_actions.yml

```
@@ -95,17 +89,42 @@ actions:
     - if (settings.skipStart != 'true'):
         - env.control.ExecDockerRunCmd [${nodes.cp.join(id,)}]
 
-  installMaven:
-    - log: "## Activate Maven on ${this}"
-    - cmd[${this}]: |-
-        mvn_path=$(find /opt/*maven*/bin -type f -name "mvn")
-        sed -i 's,#\(mvnPath =\s*\).*$,\1'$mvn_path',' /opt/tomcat/conf/digital-factory-config/jahia/jahia.properties
+  setJahiaPropertiesEnvvars:
+    - log: "## Setting jahia.properties envvars"
+    - cmd [${nodes.proc.first.id}]: '[ -d /opt/magick ] && echo "/opt/magick/bin" || echo "/usr/bin"'
+    - setGlobals:
+        jahia_cfg_imageMagickPath: ${response.out}
+    - cmd [${nodes.proc.first.id}]: echo $(ls -d /opt/*maven*/bin/mvn)
+    - setGlobals:
+        jahia_cfg_mvnPath: ${response.out}
+    - setGlobals: # Dev values by default
+        jahia_cfg_org_jahia_ehcachemanager_maxBytesLocalHeap: ${globals.org_jahia_ehcachemanager_maxBytesLocalHeap_dev}
+        jahia_cfg_org_jahia_ehcachemanager_big_maxBytesLocalHeap_cp: ${globals.org_jahia_ehcachemanager_big_maxBytesLocalHeap_dev}
+        jahia_cfg_org_jahia_ehcachemanager_big_maxBytesLocalHeap_proc: ${globals.org_jahia_ehcachemanager_big_maxBytesLocalHeap_dev}
+    - if (settings.mode == "production"):
+        - setGlobals:
+            jahia_cfg_org_jahia_ehcachemanager_maxBytesLocalHeap: ${globals.org_jahia_ehcachemanager_maxBytesLocalHeap_prod}
+            jahia_cfg_org_jahia_ehcachemanager_big_maxBytesLocalHeap_cp: ${globals.org_jahia_ehcachemanager_big_maxBytesLocalHeap_prod_cp}
+            jahia_cfg_org_jahia_ehcachemanager_big_maxBytesLocalHeap_proc: ${globals.org_jahia_ehcachemanager_big_maxBytesLocalHeap_prod_proc}
```

```suggestion
    - if (settings.mode == "production"):
        - setGlobals:
            jahia_cfg_org_jahia_ehcachemanager_maxBytesLocalHeap: ${globals.org_jahia_ehcachemanager_maxBytesLocalHeap_prod}
            jahia_cfg_org_jahia_ehcachemanager_big_maxBytesLocalHeap_cp: ${globals.org_jahia_ehcachemanager_big_maxBytesLocalHeap_prod_cp}
            jahia_cfg_org_jahia_ehcachemanager_big_maxBytesLocalHeap_proc: ${globals.org_jahia_ehcachemanager_big_maxBytesLocalHeap_prod_proc}
    - else:
        - setGlobals:
            jahia_cfg_org_jahia_ehcachemanager_maxBytesLocalHeap: ${globals.org_jahia_ehcachemanager_maxBytesLocalHeap_dev}
            jahia_cfg_org_jahia_ehcachemanager_big_maxBytesLocalHeap_cp: ${globals.org_jahia_ehcachemanager_big_maxBytesLocalHeap_dev}
            jahia_cfg_org_jahia_ehcachemanager_big_maxBytesLocalHeap_proc: ${globals.org_jahia_ehcachemanager_big_maxBytesLocalHeap_dev}
```

will sound more readable to me

---

Fixed